patches/chrome-browser-page_load_metrics-page_load_metrics_initialize.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc b/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
+index 9bb1490b5d45eb7ed208f1f6b0332a1e3732aa29..5667b397f38abae2d3f4a54a004769746c644ac1 100644
+--- a/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
++++ b/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
+@@ -62,6 +62,8 @@
+ #include "extensions/common/constants.h"
+ #endif
+ 
++#include "brave/browser/brave_performance_observers.h"
++
+ namespace chrome {
+ 
+ namespace {
+@@ -175,6 +177,7 @@ void PageLoadMetricsEmbedder::RegisterEmbedderObservers(
+       SecurityStatePageLoadMetricsObserver::MaybeCreateForProfile(
+           web_contents()->GetBrowserContext()));
+   tracker->AddObserver(std::make_unique<DataUseMetricsObserver>());
++  brave::RegisterPerformanceObservers(tracker);
```

Comment:
we can do a chomium_src override for page_load_metrics::PageLoadMetricsEmbedderBase instead of patching here